web/src/utils/mixins.js

```diff
@@ -24,6 +26,48 @@ const loadDataset = {
   },
 };
 
+/**
+ * Expects `id` to exist on the wrapping component
+ * @param {String} plotName name of the plot to subscribe to
+ */
+const plotDataMixin = plotName => ({
```

Comment:
why do you prefer

```js
const xx = {...};

export {
 xx
}
```

over 

```js
export const xx = {..}
```

Reply:
I like to be able to look at all the exported symbols from a file in one place.  I would probably have written `export const` if it were a constant instead of a function.  

However, I've never really thought too much about it.   Is there a reason to do it the other way?

Reply:
no big reason, just out of curiosity. imo the advantage is that if you add a function, you don't have to edit two places